File: src/main/java/com/youtube/tracker/TrackerApplication.java
package com.youtube.tracker;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.openfeign.EnableFeignClients;

import java.time.Instant;
import java.time.format.DateTimeFormatter;

@SpringBootApplication
@EnableFeignClients
public class TrackerApplication {
	public static void main(String[] args) {
		System.out.println("timeeee  "+DateTimeFormatter.ISO_INSTANT.format(Instant.now()));
		SpringApplication.run(TrackerApplication.class, args);
	}
}

File: src/main/kotlin/com/youtube/tracker/TrackerApplication.kt
package com.youtube.tracker

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.openfeign.EnableFeignClients
import org.springframework.context.annotation.Bean
import java.time.Clock

@SpringBootApplication
@EnableFeignClients
class TrackerApplication {
    @Bean
    fun globalClock() = Clock.systemUTC()
}

fun main(args: Array<String>) {
    runApplication<TrackerApplication>(*args)
}
